10-update_grub.conf

```diff
@@ -0,0 +1,2 @@
+[Service]
+ExecStartPost=update-grub
```

Comment:
ExecStartPost=update-grub
update-grub is an alias of grub-mkconfig ?
On some distributions, this alias does not exist, especially archlinux ...
Could you replace this command with a less specific command?
e.g grub-mkconfig -o /boot/grub/grub.cfg

Reply:
Sorry, I presumed that it was essentially universal since I can't recall it being missing on any of the distros I've used. I've fixed it now.

I've also added more detail to the README section: I previously forgot to mention the need to run `systemctl daemon-reload`.